lib/bundler/adder.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require "net/http"
+require "yaml"
+
+module Bundler
+  class Adder
+    def initialize(name, version)
+      @name = name
+      @version = version || last_version_number
+    end
+
+    def add
```

Comment:
I think all the current Bundler command classes follow an `initialize` + `run` pattern that's invoked like so: https://github.com/bundler/bundler/blob/master/lib/bundler/cli.rb#L175,  https://github.com/bundler/bundler/blob/master/lib/bundler/cli.rb#L204, etc.